ci/features/buildpackless.feature

```diff
@@ -3,7 +3,9 @@ Feature: profile d scripts without the buildpack
 
   @BUILD_DIR
   Scenario: Populates environment with secrets from Conjur
-    Given the compile script is run against the app's root folder
+    Given the build directory has a secrets.yml file
+    And VCAP_SERVICES contains cyberark-conjur credentials
+    And the supply script is run against the app's root folder
```

Comment:
Should we also have two tests that validate what happens when
- the build dir does not have secrets.yml, and/or
- VCAP_SERVICES does not contain cyberark-conjur creds?

Reply:
These conditions are already tested in the `supply.feature` so I think we're okay.